internal/dao/helm_chart.go

```diff
@@ -145,8 +145,21 @@ func (h *HelmChart) Uninstall(path string, keepHist bool) error {
 
 // ensureHelmConfig return a new configuration.
 func ensureHelmConfig(flags *genericclioptions.ConfigFlags, ns string) (*action.Configuration, error) {
+	settings := &genericclioptions.ConfigFlags{
+		Namespace:        &ns,
+		Context:          flags.Context,
+		BearerToken:      flags.BearerToken,
+		APIServer:        flags.APIServer,
+		CAFile:           flags.CAFile,
+		KubeConfig:       flags.KubeConfig,
+		Impersonate:      flags.Impersonate,
+		Insecure:         flags.Insecure,
+		TLSServerName:    flags.TLSServerName,
+		ImpersonateGroup: flags.ImpersonateGroup,
+		WrapConfigFn:     flags.WrapConfigFn,
+	}
 	cfg := new(action.Configuration)
```

Comment:
Hi @derailed, unfortunately I'm not too familiar with k9s' and Helms' codebase, so I've just been trying to orient myself on [how helm constructs their client](https://github.com/helm/helm/blob/main/pkg/cli/environment.go#L115-L136).

Just be clear: You are suggesting to do this instead?
```suggestion
func ensureHelmConfig(flags *genericclioptions.ConfigFlags, ns string) (*action.Configuration, error) {
	settings := &genericclioptions.ConfigFlags{
		Namespace:        &ns,
	}
	cfg := new(action.Configuration)
```

Reply:
@thorbenbelow nw Thank you Thorben! right I think flags.Namespace = &ns should be all we need.

Reply:
@thorbenbelow Thank you Thorben! I think the core issue here is helm does not use the passed in ns and create their own api client via a factory. Let's keep your original impl for the time being and make a copy of the flags prior to calling the helm api which I think is the safest course of action.

Reply:
Sorry for the late reply. I squashed my changes and rebased.
Do we need tests for this or is there anything else needed to make this PR mergeable?